src/Module.php

```diff
@@ -155,7 +156,13 @@ final class Module extends \luya\admin\base\Module implements CoreModuleInterfac
      * @since 1.2.0
      */
     public $userIdleTimeout = 1800;
-    
+
+    /**
+     * @var Connection The component/connection name from application.
```

Comment:
@boehsermoe in phpdocs you should use \yii\db\Connection unless the class is used in context of course.

Reply:
Type was wrong anyway. It is a string not a \yii\db\Connection.